index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header class="menu-container">
+        <nav class="navbar">
+            <div>
+            <h1 class="logo">My Logo</h1>
+           </div>
+            <div>
+          <img src="./images/Union.png" alt="menu">
+            </div>
+        </nav>
+    </header>
```

Comment:
- [ ] Please make sure to use flexbox for the layout of this header section, as required in the project requirement :point_down: Flexbox is a crucial skill to master and is important part of front end design it helps alot in responsive web development
![image](https://user-images.githubusercontent.com/97521557/234853442-0abdac5a-2a5d-42d2-978a-bf6c33704943.png)

- [ ] [OPTIONAL] I would suggest that rather than putting all the elements in the `nav` tag, you can put logo in link tag and hamburger menu in the `nav` tag